cpp/velox/benchmarks/QueryBenchmark.cc

```diff
@@ -26,7 +26,8 @@ auto BM = [](::benchmark::State& state,
              const std::vector<std::string>& datasetPaths,
              const std::string& jsonFile,
              const std::string& fileFormat) {
-  const auto& filePath = getExampleFilePath("plan/" + jsonFile);
+  // const auto& filePath = getExampleFilePath("plan/" + jsonFile);
+  const auto& filePath = "/home/morui/Downloads/gluten/cpp/velox/benchmarks/data/plan/select.json";
```

Comment:
the path contains your name, use `env-variable` or `relative-path` instead?

Reply:
Sure. It's only a draft now, will continue on the improvement.

Reply:
Fixed.